### ISSUE TITLE
Cop idea: Keyword Arguments vs Optional Arguments

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12352]
## Description

Prefer keyword arguments over optional arguments.

https://rubystyle.guide/#keyword-arguments-vs-optional-arguments

## Describe the solution you'd like

```ruby
# bad
def some_method(a, b = 5, c = 1)
  # body omitted
end

# good
def some_method(a, b: 5, c: 1)
  # body omitted
end
```

**Comment by cllns** [comment]
For reference, this exists for booleans https://www.rubydoc.info/gems/rubocop/1.27.0/RuboCop/Cop/Style/OptionalBooleanParameter

**Comment by cllns** [comment]
I think this is still worth considering.
